testing/run.sh

```diff
@@ -26,4 +26,6 @@ python -m kubeflow.testing.run_e2e_workflow \
 	--cluster=kubeflow-testing \
 	--bucket=kubernetes-jenkins \
 	--component=workflows \
-	--app_dir=/src/kubeflow/kubeflow/testing/workflows
+	--app_dir=/src/kubeflow/kubeflow/testing/workflows \
```

Comment:
Just check if prow_config.yaml exists and if it doesn't invoke the current command and if it does invoke the new command.

Then you can rebuild the image and the image will be backwards compatible. Then you don't depend on any changes  to kubeflow/testing.